scss/_tags.scss

```
@@ -0,0 +1,159 @@
+// stylelint-disable function-disallowed-list
+
+.tag {
+  // scss-docs-start tag-css-vars
+  --#{$prefix}tag-gap: #{$tag-gap};
+  --#{$prefix}tag-padding-x: #{$tag-padding-x};
+  --#{$prefix}tag-padding-y: #{$tag-padding-y};
+  --#{$prefix}tag-margin: 0;
+  @include rfs($tag-font-size, --#{$prefix}tag-font-size);
+  --#{$prefix}tag-font-shift: #{$tag-font-shift};
+  --#{$prefix}tag-font-weight: #{$tag-font-weight};
+  --#{$prefix}tag-font-spacing: #{$tag-font-spacing};
+  --#{$prefix}tag-icon-size: #{$tag-icon-size};
+  --#{$prefix}tag-icon-margin-start: #{$tag-icon-margin-start};
+  --#{$prefix}tag-close-size: #{add(var(--#{$prefix}tag-font-size), .375rem)};
+  --#{$prefix}tag-close-margin-end: #{$tag-close-margin-end};
+  --#{$prefix}tag-close-margin-start: #{$tag-close-margin-start};
+  --#{$prefix}tag-border-width: #{$tag-border-width};
+  --#{$prefix}tag-border-radius: #{$tag-border-radius};
+  --#{$prefix}tag-border-color: #{$tag-border-color};
+  --#{$prefix}tag-active-color: #{$tag-active-color};
+  --#{$prefix}tag-active-decoration-color: #{$tag-active-decoration-color};
+  --#{$prefix}tag-disabled-color: #{$tag-disabled-color};
+  --#{$prefix}tag-outline-color: var(--#{$prefix}link-hover-color);
+  // scss-docs-end tag-css-vars
+
+  display: inline-flex;
+  gap: var(--#{$prefix}tag-gap);
+  align-items: center;
+  padding: subtract(var(--#{$prefix}tag-padding-y), calc(var(--#{$prefix}tag-font-shift) * .5)) var(--#{$prefix}tag-padding-x) add(var(--#{$prefix}tag-padding-y), calc(var(--#{$prefix}tag-font-shift) * .5));
+  margin: var(--#{$prefix}tag-margin);
+  @include font-size(var(--#{$prefix}tag-font-size));
+  font-weight: var(--#{$prefix}tag-font-weight);
+  line-height: add(var(--#{$prefix}tag-font-size), .125rem);
+  color: inherit;
+  letter-spacing: var(--#{$prefix}tag-font-spacing);
+  background-color: transparent;
+  border: var(--#{$prefix}tag-border-width) solid var(--#{$prefix}tag-border-color);
+  outline-offset: 2px;
+  @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
```

```suggestion
  @include border-radius(var(--#{$prefix}tag-border-radius));
```
Not possible like this?
If so, several modifications to do in the file.